recipe/meta.yaml

```diff
@@ -1,52 +1,58 @@
 {% set name = "libsemigroups_pybind11" %}
-{% set version = "0.1.1" %}
+{% set version = "1.0.1" %}
 
 package:
-  name: libsemigroups_pybind11
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/libsemigroups/libsemigroups_pybind11/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: a51878de09443d9e632a0c502aaa47b314a54ee793731c20f1874896890dfecb
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/libsemigroups_pybind11-{{ version }}.tar.gz
+  sha256: 3fc2dca669fe4468d524d293099df5dd84074b6049f88ce222ea560b36f5bdec
 
 build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
   skip: true  # [win]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
   host:
-    - python
+    - setuptools >=64
+    - setuptools-scm >=8
+    - libsemigroups >=3.1.3
+    - packaging
     - pip
-    - libsemigroups
-    - pkgconfig >=0.29.2
-    - packaging >=20.4
-    - pybind11 >=2.6
+    - pkgconfig
+    - pybind11
+    - python
   run:
+    - numpy
     - python
-    - libsemigroups >=2.0.2
-    - pybind11 >=2.6
+    - python-graphviz
+    - typing_extensions
 
 test:
   imports:
     - libsemigroups_pybind11
+  commands:
+    - pip check
+    - pytest
   requires:
-    - pytest=6.2.4
+    - python 3.9
```

Comment:
Why are you depending on a specific python version here? It doesn't look like the recipe is `python: noarch`

Reply:
@h-vetinari thanks for getting back to us, 3.9 is the minimum version of python required by `libsemigroups_pybind11`, which is I suppose why this is as it is. Python is required to run the tests, do you have a proposed fix?

Reply:
You've already specified python as a runtime dependency of the package (and you're building for several python version separately). So you don't need to further constrain this at all
```suggestion
```